tutorial/en.md

```diff
@@ -522,114 +520,56 @@ let appState = #{
 };
 ```
 
-What we can do is modify our example so that the mutable parts are wrapped in boxes:
+What we can do is modify our example so that the mutable parts are referenced with a WeakMap:
 
 ```js
+// temporary: until WeakMaps can contain symbols as keys, we need to swap them for Maps
+globalThis.WeakMap = globalThis.Map;
```

Comment:
This screams "big memory leaks for whoever copies these examples"; can we just use `new Map` for now and move the comment that it's temporary to the `RefsRegistry` definition line?